climakitae/core/data_export.py

```diff
@@ -52,15 +53,21 @@ def _estimate_file_size(data, format):
         est_file_size = data_size + buffer_size
 
     elif format == "CSV":
-        pass  # TODO: estimate CSV file size
-
+        # Rough estimate of the number of chars per CSV line
+        # Will overestimate uncompressed size by 10-20%
+        chars_per_line = 150
+
+        if isinstance(data, xr.core.dataarray.DataArray):
+            est_file_size = np.prod(data.shape) * chars_per_line
```

Comment:
I did a little more digging and you can actually use `data.size` instead of `np.prod(data.shape)` if you wanted (same result).

Reply:
Aha, see you learn something new every day! Thx!

Reply:
Fixed in 636de95